src/website/src/pages/ec/core-template/page-header/docs/code.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Showcase
+order: 2
+---
+
+import { Playground, Html } from '@ecl/website-components';
+
+import {
+  pageHeaderTitle,
+  pageHeaderMetaTitle,
+  pageHeaderMetaTitleDescription,
+  pageHeaderBackgroundImage,
+} from '../demo';
+
+## Title
+
+<Playground
+  system="ec"
+  selectedKind="components-page-headers-page-header-core"
```

Comment:
Here you adapted this to the story name, but it's better doing the opposite, i think, i mean name the story Core and then here it will be components-page-headers-core like it is in v2